app/forms/totp_verification_form.rb

```diff
@@ -29,9 +30,10 @@ def totp_code_length
     TwoFactorAuthenticatable::OTP_LENGTH
   end
 
-  def extra_analytics_attributes
+  def extra_analytics_attributes(cfg_id)
     {
       multi_factor_auth_method: 'totp',
+      mfa_id: cfg_id,
```

Comment:
same, can we make this `totp_config_id` or prefix the value like `"totp-#{cfg_id}"`